example/device/framegrabber/getRealSense2Info.cpp

```diff
@@ -50,7 +50,11 @@ int main()
     vpRealSense2 rs;
     rs.open();
     std::cout << "RealSense characteristics:\n" << rs << std::endl;
-    std::cout << "Depth scale: " << std::setprecision(std::numeric_limits<float>::max_digits10) << rs.getDepthScale() << std::endl;
+
+    std::string product = rs.getProductLine();
+
+    if(product.compare("D400") == 0) // If it has depth sensor
```

Comment:
There is also the SR300 camera that we have in the lab and that has depth info.

If would prefer to change like:
```
if(product.compare("T200") != 0) {
  std::cout << "Depth scale: " << std::setprecision(std::numeric_limits<float>::max_digits10) << rs.getDepthScale() << std::endl;
}
```